cmd/pack3d/main.go

```diff
@@ -1,33 +1,34 @@
 /*
 Instruction: write down this into the command line to use the software
-<pack3d {your_frame_size_X,your_frame_size_Y,your_frame_size_Z} spacing your_output_name STL_numbers STL_path STL_numbers STL_path .....>
-For example: <pack3d {100,100,100} 5 Pikacu 1 /home/corner.stl 2 /home/Pika.stl>
+<pack3d --build_volume=your_frame_size_X,your_frame_size_Y,your_frame_size_Z --json_file=json_config_file --spacing=spacing --filename=export_filename>
+For example: <pack3d --json_file=input.json --build_volume=100,100,100>
 The unit of frame and spacing is millimeters.
 */
 
 package main
 
 import (
 	"encoding/json"
+	"flag"
 	"fmt"
 	"io/ioutil"
```

Comment:
```suggestion
      "gonum"
	"io/ioutil"
```

Reply:
This library is needed to do the inverse of a matrix and to multiply two matrices.

https://pkg.go.dev/gonum.org/v1/gonum/mat#section-documentation

https://medium.com/wireless-registry-engineering/gonum-tutorial-linear-algebra-in-go-21ef136fc2d7

Reply:
It will automatically get imported if we use this library in the file.

Reply:
Yes but as the code is meant to run in Docker containers and this dependency's name (and version number) need to be explicitly added in.